index/templates/index.html

```diff
@@ -4,15 +4,14 @@
 {% block content %}
 <div class="jumbotron jumbotron-fluid">
     <div class="caption">
-        <h1 class="pontiac">Student Senate Petitions</h1>
-        <h4>Create and sign petitions.</h4>
-        <h4>Make your voice heard by the RPI Student Senate and the campus community.</h4>
+        <h1 class="pontiac">Let Your Voice Be Heard!</h1>
+        <br><br><br>
```

Comment:
I like the wording change!
I think you could add "Student Senate Petitions" as maybe an `<h4>` after this line, and I think 3 breaks might be too much space.

Reply:
Tried adding the "Student Senate Petitions". While the desktop version looks good, the mobile version has text spilling into the rest of the page.
![image](https://user-images.githubusercontent.com/32110950/56161957-dd102000-5f98-11e9-8cb7-c6fc9eac9dbc.png)

Reply:
Took what you suggested and added a little more to the splash page. Result looks like this:
![image](https://user-images.githubusercontent.com/32110950/56163555-98868380-5f9c-11e9-8507-b397839c05fc.png)
However, the "Recent Petitions" are still off-center